tests/CMakeLists.txt

```
@@ -107,6 +103,10 @@ if (MSVC AND CMAKE_SIZEOF_VOID_P EQUAL 4)
     target_link_options(vk_layer_validation_tests PRIVATE /LARGEADDRESSAWARE)
 endif()
 
+if(MSVC_IDE)
+    set_target_properties(vk_layer_validation_tests PROPERTIES VS_DEBUGGER_ENVIRONMENT "VK_LAYER_PATH=$(SolutionDir)\\layers\\$(Configuration)")
```

More robust to use generator expressions in this situation since `VS_DEBUGGER_ENVIRONMENT` [supports](https://cmake.org/cmake/help/latest/prop_tgt/VS_DEBUGGER_ENVIRONMENT.html?highlight=vs_debugger_environment) it.

```suggestion
    set_target_properties(vk_layer_validation_tests PROPERTIES VS_DEBUGGER_ENVIRONMENT "VK_LAYER_PATH=$<TARGET_FILE_DIR:VkLayer_khronos_validation>")
```

---

Thanks for the suggestion!